### PR TITLE
Fix getConversation getting user_conversation instead of conversation

### PR DESCRIPTION
**Pull request by chankiyu22**

### CODE REVIEW

File: js-sdk/demo/bundle.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ function SkygearChatContainer() {
     query.transientInclude('conversation');
     return skygear.publicDB.query(query).then(function(records) {
       if (records.length > 0) {
-        return records[0];
+        return records[0]._transient.conversation;
```
Comment (rickmak) [inline js-sdk/demo/bundle.js]: Please use `$transient`

**Comment by rickmak** [comment]
In normal conversation display, we will use the user_conversation to store and display attribute like `unread_count`. So it is intended behaviour that getConversation is returing `user_conversation`.

The fix to #31 should be better done by making the `addParticipants` accepting the `UserConversation` object?

**Comment by chankiyu22** [comment]
Yes, I overlooked the `unread_count` attribute from user_conversation object, so let it keep retuning `user_conversation`.

I think the problem is that the operations `deleteConversation`, `updateConversation`, `addParticipants`, `removeParticipants`, `addAdmins` and `removeAdmins` are operating on `user_conversation` object which is not intended. So I make changed to those operations to update on `conversation` object by getting the `conversation` transient from `user_conversation` object.

**Comment by rickmak** [comment]
Pick as 94d7c07

**Pull request closed without merging**